nesc/whip6/platforms/parts/mcu/cc26xx/uart/HalUARTReadNowPrv.nc

```diff
@@ -60,6 +61,7 @@ implementation {
             atomic busy = FALSE;
             signal ReadNow.readDone(SUCCESS, (uint8_t)value);
         }
-
     }
+
+    default async event void ReadNow.readDone(error_t result, uint8_t val) { }
```

Comment:
Same here. Why do you want a default event handler in a module that provides only a single interface?